deployment/README.md

```diff
@@ -286,12 +286,11 @@ should be sufficient.
 ### Fluentd
 
 Fluentd is deployed with no replicas. Once you have ElasticSearch
-running as desired, scale fluentd to every node to feed logs into ES.
-The example below would be for a cluster with 3 nodes; substitute the
-actual number of nodes in your cluster.
+running as desired, label the nodes to deploy fluentd to in order to feed logs
+into ES. The example below would be to label a node named
+'ip-172-18-2-170.ec2.internal'.
 
-    $ oc scale dc/logging-fluentd --replicas=3
-    $ oc scale rc/logging-fluentd-1 --replicas=3
+    oc label node/ip-172-18-2-170.ec2.internal logging-infra=fluentd
 
 Kubernetes is developing a method of scheduling a pod to dynamically run
```

Comment:
We need to evaluate this paragraph as it looks like this change counters this statement